pkg/plugins/policies/meshtrace/api/v1alpha1/validator.go

```diff
@@ -83,8 +83,13 @@ func validateBackend(conf Conf, backendsPath validators.PathBuilder) validators.
 	var verr validators.ValidationError
 	backend := pointer.Deref(conf.Backends)[0]
 	firstBackendPath := backendsPath.Index(0)
-	if (backend.Datadog != nil) == (backend.Zipkin != nil) {
-		verr.AddViolationAt(firstBackendPath, validators.MustHaveOnlyOne("backend", "datadog", "zipkin"))
+
+	switch {
+	case backend.Datadog != nil:
+	case backend.Zipkin != nil:
+	case backend.OpenTelemetry != nil:
+	default:
+		verr.AddViolationAt(firstBackendPath, validators.MustHaveOnlyOne("backend", "datadog", "zipkin", "openTelemetry"))
```

Comment:
Unfortunately I think something like the following is the only correct way to specify "exactly one is defined", until `controller-tools` supports `oneOf` and we can avoid writing code like this

```suggestion
	var defined int
	if backend.Datadog != nil {
		defined++
	}
	if backend.Zipkin != nil {
		defined++
	}
	if backend.OpenTelemetry != nil {
		defined++
	}
	if defined != 1 {
		verr.AddViolation("", validators.MustHaveOnlyOne("backend", "datadog", "zipkin", "openTelemetry"))
```